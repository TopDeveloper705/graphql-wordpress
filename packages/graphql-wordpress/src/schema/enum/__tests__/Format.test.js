import FORMAT from 'enum/Format';

describe('Test enum values', () => {
  test('FORMAT values', () => {
    expect(FORMAT.getValues()).toMatchSnapshot();
  });
});
