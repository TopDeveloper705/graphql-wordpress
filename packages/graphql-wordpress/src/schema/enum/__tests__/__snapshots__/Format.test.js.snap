// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test enum values FORMAT values 1`] = `
Array [
  Object {
    "deprecationReason": undefined,
    "description": undefined,
    "isDeprecated": false,
    "name": "STANDARD",
    "value": "standard",
  },
  Object {
    "deprecationReason": undefined,
    "description": undefined,
    "isDeprecated": false,
    "name": "ASIDE",
    "value": "aside",
  },
  Object {
    "deprecationReason": undefined,
    "description": undefined,
    "isDeprecated": false,
    "name": "CHAT",
    "value": "chat",
  },
  Object {
    "deprecationReason": undefined,
    "description": undefined,
    "isDeprecated": false,
    "name": "GALLERY",
    "value": "gallery",
  },
  Object {
    "deprecationReason": undefined,
    "description": undefined,
    "isDeprecated": false,
    "name": "LINK",
    "value": "link",
  },
  Object {
    "deprecationReason": undefined,
    "description": undefined,
    "isDeprecated": false,
    "name": "IMAGE",
    "value": "image",
  },
  Object {
    "deprecationReason": undefined,
    "description": undefined,
    "isDeprecated": false,
    "name": "QUOTE",
    "value": "quote",
  },
  Object {
    "deprecationReason": undefined,
    "description": undefined,
    "isDeprecated": false,
    "name": "STATUS",
    "value": "status",
  },
  Object {
    "deprecationReason": undefined,
    "description": undefined,
    "isDeprecated": false,
    "name": "VIDEO",
    "value": "video",
  },
  Object {
    "deprecationReason": undefined,
    "description": undefined,
    "isDeprecated": false,
    "name": "AUDIO",
    "value": "audio",
  },
]
`;
